docs/conf.py

```diff
@@ -57,9 +57,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0.0'
+version = u'4.0.0'
 # The full version, including alpha/beta/rc tags.
-release = u'1.0.0'
+release = u'4.0.0'
```

Comment:
I think these affect the version that appears here:

<img width="249" alt="Screenshot 2021-06-11 at 23 35 53" src="https://user-images.githubusercontent.com/34811569/121755023-fe83dc80-cb0d-11eb-87d0-5052a7e8ed30.png">

This should be fairly in sync with the releases otherwise it might lead someone to believe that the documentation is out of date

Reply:
Defo.